common/djangoapps/third_party_auth/tests/test_pipeline.py

```diff
@@ -78,7 +78,7 @@ def setUp(self):
         ('S.K.', 'S_K', False),
         ('S.K.', 'S_K_-9fe2', True),
         ('usernamewithcharacterlengthofmorethan30chars', 'usernamewithcharacterlengthofm', False),
-        ('usernamewithcharacterlengthofmorethan30chars', 'usernamewithch-9fe2', True),
+        ('usernamewithcharacterlengthofmorethan30chars', 'usernamewithcharacterlen-9fe2', True),
```

Comment:
This is still testing for the 4-digit suffix -- seems to me we should test for the new 6-digit suffix?

Reply:
it is an artifact of the test. its not a great test - but i did run the code by hand and it works properly. ive also observed how its working now in production via the logs (at 16) and its working properly.